pact/Pact/Core/IR/ModuleHashing.hs

```diff
@@ -223,11 +223,8 @@ encodeApp operator operands =
   parens $ hsep (operator:operands)
 
 encodeSchema :: Schema -> Builder
-encodeSchema (Schema sc) =
-  hsep (encodePair <$> M.toList sc)
-  where
-  encodePair (Field f, ty) =
-    T.encodeUtf8Builder f <> ":" <> encodeType ty
+encodeSchema (Schema n _sc) =
+  encodeQualName n
```

Comment:
note: this change is quite significant. It means our stable encoding of modules do not blow up nested schemas in types in size, so the output serialization stays small. Schemas are basically serialized as free variable references which is what we want.